client/state/selectors/get-site-computed-attributes.js

```diff
@@ -0,0 +1,49 @@
+/**
```

Comment:
Yes I agree I will make this re-factoring, thank you for pointing me in the right direction I will make this change.